VideoApi/VideoApi/Extensions/ContractExtensions.cs

```diff
@@ -74,5 +74,9 @@ public static Domain.Enums.LinkedParticipantType MapToDomainEnum(this Contract.E
         {
             return Enum.Parse<Domain.Enums.LinkedParticipantType>(taskType.ToString());
         }
+        public static Contract.Enums.RoomStatus MapToContractEnum(this Domain.Enums.RoomStatus taskType)
```

Comment:
Can this be grouped together with the other MapToContractEnum methods? Other than that everything looks great!